src/c_vec.rs

```diff
@@ -273,12 +228,19 @@ impl<T> CSlice<T> {
     pub fn is_empty(&self) -> bool { self.len() == 0 }
 }
 
-impl<T> AsSlice<T> for CSlice<T> {
-    /// View the stored data as a slice.
-    fn as_slice<'a>(&'a self) -> &'a [T] {
```

Comment:
I can't compile rgsl because of a Rust ICE, so I couldn't see this. Thanks ! Now the problem is to see what should I add (or not) in CSlice and/or CVec. But clearly I think this shouldn't have been deleted.